os/net/netmgr/netstack_lwip.c

```diff
@@ -427,22 +427,23 @@ static void _get_raw_info(struct netmon_sock *sock_info, struct lwip_sock *lsock
 	sock_info->remote.ip.sin_port = 0;
 }
 
-extern struct lwip_sock *get_lwip_sock_info(void);
+extern struct socketlist *get_lwip_sock_info(void);
 
 /**
  * Copy the global socket contents to the input
  *
  * @param data is used to target
  */
-extern struct lwip_sock *get_lwip_sock_info(void);
+extern struct socketlist *get_lwip_sock_info(void);
 
 static int lwip_ns_getstats(int fd, struct netmon_sock **sock_info)
 {
-	struct lwip_sock *sockets = get_lwip_sock_info();
-	if (!sockets[fd].conn) {
+	struct socketlist *list = get_lwip_sock_info();
+	struct lwip_sock *sock = (struct lwip_sock *) list->sl_sockets[fd].sock;
```

Comment:
no space here i believe b/w typecasting and list->sl_sockets[fd].sock

Reply:
done